libs/frontend/util-router/src/lib/scroll-behavior.ts

```diff
@@ -87,15 +71,14 @@ export function scrollToPosition(
       console.log(`No el with selector: ${position.el} in scrollBehavior.`)
       return
     }
-    scrollToOptions = getElementPosition(el, position)
-  } else {
-    scrollToOptions = position
-  }
 
-  if (scrollRoot) {
-    scrollRoot.scroll(scrollToOptions)
+    el.scrollIntoView()
```

Comment:
Hi @sampullman 

I noticed that the scroll behavior of a hash link in live mode is a little strange; the position seems to jump/switch between two places upon click.
Since this `if` section is intended to scroll to a specific element, `scrollIntoView()` seems to work well here. I may have missed something, it would be great if you could double check if this makes sense!

https://github.com/samatechtw/pubstudio-builder/assets/59676941/a8f61ef7-886e-4847-b02c-74dbdab13ff0